catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -195,6 +195,16 @@ def __init__(
 
         self.ingestion_errors: list[IngestionError] = []  # Keep track of skipped errors
 
+        environment = Variable.get(
+            "ENVIRONMENT", default_var='dev'
+        )
+
+        should_verbose_log = self.dag_id in Variable.get(
+            "SHOULD_VERBOSE_LOG", default_var=[], deserialize_json=True
+        )
+
+        self._should_verbose_log = environment != 'prod' or should_verbose_log
```

Comment:
```suggestion
        self._should_verbose_log = environment == "local" or should_verbose_log
```

Small change to be in line with #4053.